homeassistant/helpers/entity_registry.py

```diff
@@ -457,9 +457,9 @@ def __init__(self) -> None:
         super().__init__()
         self._entry_ids: dict[str, RegistryEntry] = {}
         self._index: dict[tuple[str, str, str], str] = {}
-        self._config_entry_id_index: dict[str, list[str]] = {}
-        self._device_id_index: dict[str, list[str]] = {}
-        self._area_id_index: dict[str, list[str]] = {}
+        self._config_entry_id_index: dict[str, dict[str, Literal[True]]] = {}
+        self._device_id_index: dict[str, dict[str, Literal[True]]] = {}
+        self._area_id_index: dict[str, dict[str, Literal[True]]] = {}
```

Comment:
Sometimes `None` is used for this case, but I think its nicer to have existence be truthy instead of falsey